burnman/anisotropy.py

```diff
@@ -0,0 +1,471 @@
+
+        return velocities, eigenvectors
+
+    def plot_velocities(self):
+        """
+        Makes colour plots of:
+        Compressional wave velocity: Vp
+        Anisotropy: (Vs1 - Vs2)/(Vs1 + Vs2)
+        Vp/Vs1
+        linear compressibility: beta
+        Youngs Modulus: E
+        """
+
+        try:
+            plt.style.use('ggplot')
+            plt.rcParams['axes.facecolor'] = 'white'
+            plt.rcParams['axes.edgecolor'] = 'black'
+            plt.rcParams['figure.figsize'] = 16, 10 # inches
+        except:
+            pass
+        
+        zeniths = np.linspace(np.pi/2., np.pi, 31)
+        azimuths = np.linspace(0., 2.*np.pi, 91)
+        Rs = np.sin(zeniths)/(1. - np.cos(zeniths))
+        r, theta = np.meshgrid(Rs, azimuths)
+        
+        vps = np.empty_like(r)
+        vs1s = np.empty_like(r)
+        vs2s = np.empty_like(r)
+        betas = np.empty_like(r)
+        Es = np.empty_like(r)
+        for i, az in enumerate(azimuths):
+            for j, phi in enumerate(zeniths):
+                d = np.array([np.cos(az)*np.sin(phi), np.sin(az)*np.sin(phi), -np.cos(phi)]) # change_hemispheres
+                velocities = self.wave_velocities(d)
+                betas[i][j] = self.linear_compressibility(d)
+                Es[i][j] = self.youngs_modulus(d)
+                vps[i][j] = velocities[0][0]
+                vs1s[i][j] = velocities[0][1]
+                vs2s[i][j] = velocities[0][2]
+                
+        fig = plt.figure()
+        names = ['Vp (km/s)', 'Vs1 (km/s)', 'Vp/Vs1', 'S-wave anisotropy (%)', 'Linear compressibility (GPa$^{-1}$)', 'Youngs Modulus (GPa)']
+        items = [vps/1000., vs1s/1000., vps/vs1s, 200.*(vs1s - vs2s)/(vs1s + vs2s), betas*1.e9, Es/1.e9]
+        ax = []
+        im = []
+        ndivs = 100
+        for i, item in enumerate(items):
+            ax.append(fig.add_subplot(2, 3, i+1, projection='polar'))
+            ax[i].set_yticks([100])
+            ax[i].set_title(names[i])
+
+            vmin = np.min(item)
+            vmax = np.max(item)
+            spacing = np.power(10., np.floor(np.log10(vmax - vmin)))
+            nt = int((vmax - vmin - vmax%spacing + vmin%spacing)/spacing)
+            if nt == 1:
+                spacing = spacing/4.
+            elif nt < 4:
+                spacing = spacing/2.
+            elif nt > 8:
+                spacing = spacing*2.
+                
+            tmin = vmin + (spacing - vmin%spacing)
+            tmax = vmax - vmax%spacing
+            nt = int((tmax - tmin)/spacing + 1)
+            
+            ticks = np.linspace(tmin, tmax, nt)
+            im.append(ax[i].contourf(theta, r, item, ndivs, cmap=plt.cm.jet_r, vmin=vmin, vmax=vmax))
+            lines = ax[i].contour(theta, r, item, ticks, colors=('black',), linewidths=(1,))
+            
+            cbar = fig.colorbar(im[i], ax=ax[i], ticks=ticks)
+            cbar.add_lines(lines)
+
+        plt.tight_layout()
+        plt.show()
```

Comment:
I've moved the function to the example - it's probably more appropriate than a fixed function.